pkg/report/writer.go

```diff
@@ -103,15 +104,21 @@ func (tw TableWriter) write(result Result) {
 		}
 		var row []string
 		if tw.Output == os.Stdout {
-			row = []string{v.PkgName, v.VulnerabilityID, dbTypes.ColorizeSeverity(v.Severity),
-				v.InstalledVersion, v.FixedVersion}
+			row = []string{v.PkgName, v.VulnerabilityID, dbTypes.ColorizeSeverity(v.Severity), v.InstalledVersion, v.FixedVersion}
 		} else {
 			row = []string{v.PkgName, v.VulnerabilityID, v.Severity, v.InstalledVersion, v.FixedVersion}
 		}
 
 		if !tw.Light {
 			row = append(row, title)
 		}
+
+		if strings.HasPrefix(strings.ToLower(v.VulnerabilityID), "cve") {
+			row = append(row, strings.TrimPrefix(v.URL, "https://"))
+		} else {
+			row = append(row, "")
+		}
+
```

Comment:
Also, we can just call `GetPrimaryLink` here. In that case, the URL will not be empty, an AVD link or another link.